src/entities/message.js

```diff
@@ -0,0 +1,33 @@
+const BaseEntity = require('./base')
+
+class Message extends BaseEntity {
+    set text(text) {
+        if (!text) throw new Error('Message is required!')
+        this.attr.text = text.trim()
+    }
+
+    get text() {
+        return this.attr.text
+    }
+
+    get messageRoom() {
+        return this.attr.roomId
+    }
+
+    get messageCreatedDate() {
```

Comment:
is it used? 

```javascript
message.messageCreatedDate // looks weird. tautological and the attribute is not a date, it is time. 
```

Reply:
Not used.
Apparently I was planning to use it, but then went the other way.
Removed.